data/payd/payd.go

```diff
@@ -117,6 +117,9 @@ func (p *Payd) baseURL() string {
 
 func (p *Payd) CreateUser(ctx context.Context, req models.UserDetails) (*models.User, error) {
 	var user *models.User
+	if req.Name == "" || req.Email == "" {
+		return nil, errors.New("Must include name and email for user registration.")
```

Comment:
Can start with lower case as per the rationale here https://github.com/go-lang-plugin-org/go-lang-idea-plugin/issues/2151